lib/admin.php

```diff
@@ -409,6 +418,23 @@ function linkedin() {
 		$this->checkbox( $id , __( 'Use this button?', 'genesis-simple-share' ) );
 
 	}
+
+	/**
+ 	 * Create Love settings metabox output
+ 	 *
+ 	 *
+ 	 * @since 0.1.0
+ 	 *
+ 	 */
+	function love() {
+	
+		$id = 'love';
+		
+		$this->checkbox( $id , __( 'Use this button?', 'genesis-simple-share' ) );
+		
+		//echo $this->sort_text;
```

Comment:
remove debug code (I probably need to do this throughout too but that's another issue)